index.js

```diff
@@ -476,16 +492,18 @@ function session(options) {
     if (!req.sessionID) {
       debug('no SID sent, generating session');
       generate();
+      cleanEventListeners();
```

Comment:
Why would we remove the event listeners on the store for this condition, which is that a request came in without a session cookie on it?